arm-compute/2016-03-30/swagger/compute.json

```diff
@@ -3705,6 +3705,32 @@
       },
       "description": "The instance view of a virtual machine boot diagnostics."
     },
+    "VirtualMachineIdentity": {
+      "properties": {
+        "principalId": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The principal id of virtual machine identity."
+        },
+        "tenantId": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The tenant id associated with the virtual machine."
+        },
+        "type": {
+          "type": "string",
+          "description": "The identity type.",
+          "enum": [
```

Comment:
re: `type` : 

A couple of points: 
- is there only one value for this? Are there supposed to be more in the future?
- is this value ever provided by the user, or just returned from the service?
- `The identity type` doesn't offer a lot of detail -- especially since there isn't more than one possible value-- what exactly is this conveying?